### PR TITLE
copy change to add currency to add funds header

### LINKED ISSUE #2199: Small copy change to 'Add Funds' Screen
AC:

[Figma](https://www.figma.com/file/bwIY7peeykI8K4wTrXs4iN/Fiat-Connect?node-id=940%3A16749)

* Add funds => Add [currency] in the header
* [Currency] next to the total on the right

### PR DESCRIPTION
**Pull request by lisaqyli**
### Description
fixes #2199 
Adding the currency to the header of add funds screen

### CODE REVIEW

**Comment by lisaqyli** [comment]
Screenshot_20220412_230229_org celo mobile alfajores dev

File: packages/mobile/locales/base/translation.json [hunk 1]
```diff
@@ -266,6 +266,7 @@
   "withdrawScanQrTitle": "Scan",
   "tokenBalance": "{{token}} Balance",
   "addFunds": "Add Funds",
```
Comment (satish-ravi) [inline packages/mobile/locales/base/translation.json:268]: Can this be removed? Or is this used in another place?

**Review by satish-ravi — APPROVED** [review]

File: packages/mobile/locales/base/translation.json (hunk 1 above)
Comment (jh2oman) [inline packages/mobile/locales/base/translation.json:268]: Is this string used anymore? can it be deleted?

**Review by jh2oman — APPROVED** [review]
Looks good. I think we can ignore the codecov patch on this one

**Review by cajubelt — COMMENTED** [review]
@lisaqyli what about this AC?
`[Currency] next to the total on the right`
not seeing that in the screenshot

File: packages/mobile/locales/base/translation.json (hunk 1 above)
Comment (lisaqyli) [reply packages/mobile/locales/base/translation.json:268]: yeah, it's used in other places such as FiatExchange.tsx and FiatExchangeOptions.tsx

Comment (lisaqyli) [reply packages/mobile/locales/base/translation.json:268]: yeah, it's used in other places such as FiatExchange.tsx and FiatExchangeOptions.tsx

**Comment by lisaqyli** [comment]
Screenshot_20220418_224936_org celo mobile alfajores dev

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Android internal release build , Test flight release build V1.31.0 and updated status in test rails.

**Test rail links:**
**iOS15:** https://valoraapp.testrail.io/index.php?/tests/view/30610
**iOS 14** https://valoraapp.testrail.io/index.php?/tests/view/30672

**Devices:** iPhone 13 mini(15.1.1), iPhone 12(14.7.1)

**CC:** @lisaqyli 
Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Android internal release build , Test flight release build V1.31.0 and updated status in test rails.

Observation : User is able to see add cUSD / add cEUR / add CELO text on header for add funds flow

Test rail links:

Android 11: https://valoraapp.testrail.io/index.php?/tests/view/30703

Devices: Google Pixel 2XL(11.0)

CC: @lisaqyli
Thanks..!!

**Pull request merged**